src/commands.ts

```diff
@@ -3,281 +3,285 @@ import * as vscode from 'vscode';
-    vscode.QuickPickItem[] {
-  const historicEvents = getRecentEvents(extensionContext, 20);
-
-  // Get a unique list of events and take the first 5
-  const recentEvents =
-    [...new Set<string>(historicEvents.filter((e: string) => events.includes(e)))].slice(0,5);
-
-  const remainingEvents = events.filter((e) => !recentEvents.includes(e));
-
-  const recentItems = recentEvents.map((e) => {
-    return {
-      label: e,
-      description: 'recently triggered'
-    };
-  });
-
-  const remainingItems = remainingEvents.map((e) => {
-    return {
-      label: e,
-    };
-  });
-  return [...recentItems, ...remainingItems];
-}
-
-export function openReportIssue() {
-  telemetry.sendEvent('openReportIssue');
-  const {name, publisher} = getExtensionInfo();
-
-  vscode.commands.executeCommand('vscode.openIssueReporter', {
-    extensionId: `${publisher}.${name}`,
-  });
-}
-
-export function openWebhooksDebugConfigure() {
-  vscode.commands.executeCommand('workbench.action.debug.configure');
-}
-
-export function openDocs() {
-  telemetry.sendEvent('openDocs');
-  vscode.env.openExternal(
-    vscode.Uri.parse('https://stripe.com/docs/stripe-vscode')
-  );
-}
-
-export function openSurvey() {
-  telemetry.sendEvent('openSurvey');
-  const extensionInfo = getExtensionInfo();
-
-  const query = querystring.stringify({
-    platform: encodeURIComponent(osName()),
-    vscodeVersion: encodeURIComponent(vscode.version),
-    extensionVersion: encodeURIComponent(extensionInfo.version),
-    machineId: encodeURIComponent(vscode.env.machineId),
-  });
-
-  const url = `https://stri.pe/vscode-feedback?${query}`;
-  vscode.env.openExternal(vscode.Uri.parse(url));
-}
-
-export function openTelemetryInfo() {
-  vscode.env.openExternal(
-    vscode.Uri.parse('https://code.visualstudio.com/docs/getstarted/telemetry')
-  );
-}
-
-export function resendEvent(stripeTreeItem: StripeTreeItem) {
-  telemetry.sendEvent('resendEvent');
-  terminal.execute('events', ['resend', stripeTreeItem.metadata.id]);
+    if (Array.isArray(options.events)) {
+      const invalidEventCharsRE = /[^a-z_.]/;
+      const invalidEvent = options.events.find((e: string) => invalidEventCharsRE.test(e));
+      if (invalidEvent) {
+        await vscode.window.showErrorMessage(`Invalid chararacters found in event: ${invalidEvent}. For a list of all possible events, see https://stripe.com/docs/api/events/types.`);
+        return;
+      }
+    }
+    const forwardToFlag = forwardTo
+      ? ['--forward-to', forwardTo]
+      : [];
+
+    const forwardConnectToFlag = forwardConnectTo
+      ? ['--forward-connect-to', forwardConnectTo]
+      : [];
+
+    const eventsFlag = Array.isArray(options.events) && options.events.length > 0
+      ? ['--events', options.events.join(',')]
+      : [];
+
+    this.terminal.execute('listen', [...forwardToFlag, ...forwardConnectToFlag, ...eventsFlag]);
+  };
+
+  openLogsStreaming = () => {
```

Comment:
Another note: Most of these are changing function key words so hiding whitespace changes in the review view will be helpful